taccsite_cms/custom_app_settings.example.py

```diff
@@ -0,0 +1,3 @@
+CUSTOM_APPS = ['apps.custom_example']
+CUSTOM_MIDDLEWARE = []
+STATICFILES_DIRS = ()
```

Comment:
Is there a way for this file to also import settings values into the django settings object?

Reply:
Not sure I understand the question- are you saying you'd prefer to do `from taccsite_cms.custom_app_settings import *` in the main settings file and mutate the arrays/tuples inside this file instead? Otherwise, there are already ways to override settings using `secrets` or `settings_custom`.

Reply:
Sorry, I forgot there was a settings_custom.